examples/resources/bloxone_dns_view/resource.tf

```diff
@@ -3,12 +3,12 @@ resource "bloxone_ipam_ip_space" "example_ip_space" {
   name = "example_ip_space"
 }
 
-resource "bloxone_ipam_ip_space" "example_dns_view" {
-  name = "example_ip_space"
+resource "bloxone_dns_view" "example_dns_view" {
```

Comment:
the `_dns_view` in `example_dns_view` is redundant and you can remove it. It can be used if there is more than one resource of same type and we want to differentiate between them

Same comment applies to `example_ip_space`

Reply:
fixed